csrc/scheduler/matmul_hopper+.cpp

```diff
@@ -926,6 +923,122 @@ void HopperPlus::scheduleEpilogueWithSmemEpilogue() {
   }
 }
 
+constexpr int64_t hardcoded_smem_vectorize_factor = 4;
```

Comment:
Do we want to move this constant to the top or more central location?

Reply:
Moved this and `hardcoded_blackwell_splitk_vectorization_factor` up.